python-recipes/semantic-cache/semantic_caching_gemini.ipynb

```diff
@@ -93,11 +100,24 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 10,
+      "execution_count": 1,
       "metadata": {
         "id": "-fR3mWQvdKxb"
       },
-      "outputs": [],
+      "outputs": [
+        {
```

Comment:
We will need to undo this one to get rid of the error print out below committed to the notebook source. Should be easy to just add back in the nbval skip after